src/codegen/runtime_functions.cpp

```diff
@@ -71,6 +73,12 @@ storage::TileGroup *RuntimeFunctions::GetTileGroup(storage::DataTable *table,
   return tile_group.get();
 }
 
+storage::TileGroup *RuntimeFunctions::GetTileGroupByGlobalId(
```

Comment:
Change the name `GetTileGroupByGlobalId` to `GetTileGroupById` to make the API names consistent.